From f53b7938457f2e08bac09d71c507b5043abe3bba Mon Sep 17 00:00:00 2001
From: Marcel Metz <mmetz@adrian-broher.net>
Date: Wed, 9 Mar 2016 14:07:23 +0100
Subject: [PATCH] CMake: Add 'lib' prefix to output files on Windows

This mimics the current shared library naming configuration of the MSVC
project files.
---
 lib/CMakeLists.txt | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/lib/CMakeLists.txt b/lib/CMakeLists.txt
index eba59c9..c0ed621 100644
--- a/lib/CMakeLists.txt
+++ b/lib/CMakeLists.txt
@@ -73,6 +73,12 @@ if (NOT BUILD_FRAMEWORK)
     add_library(vorbisenc ${VORBISENC_SOURCES})
     add_library(vorbisfile ${VORBISFILE_SOURCES})
 
+    set_target_properties(
+        vorbis vorbisenc vorbisfile PROPERTIES
+        IMPORT_PREFIX "lib"
+        PREFIX "lib"
+    )
+
     get_version_info(VORBIS_VERSION_INFO "V_LIB_CURRENT" "V_LIB_AGE" "V_LIB_REVISION")
     set_target_properties(vorbis PROPERTIES SOVERSION ${VORBIS_VERSION_INFO})
     get_version_info(VORBISENC_VERSION_INFO "VE_LIB_CURRENT" "VE_LIB_AGE" "VE_LIB_REVISION")
-- 
2.7.4

